### PR TITLE
feat(worker): add logging

### PR DESCRIPTION
**Pull request by Realtin**
As we don't have a repoSlug for most of these jobs, this will only work with https://github.com/greenkeeperio/gk-log/pull/4  so the repoSlug is considered optional in the initialization of the logs.

### CODE REVIEW

**Comment by janl** [comment]
would a repoSlug be available in the `job.data` for certain jobs?

**Comment by Realtin** [comment]
not for all the jobs!  payment jobs are only account focused and don't have reposlugs

**Comment by Realtin** [comment]
don't forget to update greenkeeper to the newest version of gk-logs!

**Comment by janl** [comment]
> but most of them have it somewhere like data.repository.fullname

right, would be nice if we could use it when it is there.

**Comment by janl** [comment]
e.g. insatiate `Log` with `repoSlug: job.data.repository.fullname || null` or something like that.

File: lib/worker.js [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
 
 module.exports = async function worker (scheduleJob, channel, job) {
   const data = JSON.parse(job.content.toString())
   if (dev) console.log(data.name, data.type, data.action)
+  const { logs } = await dbs()
+  const accountId = Number(data.accountId) ||
+    _.get(data, 'repository.owner.id') ||
+    _.get(data, 'installation.account.id') ||
+    _.get(data, 'organization.id') ||
+    null
```
Comment (janl) [inline lib/worker.js]: now that `accountId` can be `null`, too https://github.com/greenkeeperio/gk-log/pull/4/files needs to be adapted to account for that.

Comment (Realtin) [reply lib/worker.js]: true: https://github.com/greenkeeperio/gk-log/pull/5

**Pull request merged**